core/block/import/types.go

```diff
@@ -37,5 +36,5 @@ type Creator interface {
 // IDGetter is interface for updating existing objects
 type IDGetter interface {
 	//nolint:lll
-	Get(ctx *session.Context, cs *converter.Snapshot, sbType sb.SmartBlockType, createdTime time.Time, updateExisting bool, oldIDToNew map[string]string) (string, treestorage.TreeStorageCreatePayload, error)
```

Comment:
Remove not used `oldIDToNew` parameter in function. We can get `sbType` from `converter.Snapshot`